bindings/pydeck/pydeck/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.1"
+__version__ = '0.7.2'
```

Comment:
This doesn't look like correct `black` formatting. You should run `make pre-commit` in the local dir: https://github.com/visgl/deck.gl/blob/e1b18d42245922714bf6752ff1819f3279cbc39a/bindings/pydeck/Makefile#L27-L28

Reply:
@ajduberstein we should probably add `black` to CI; it doesn't look like it's currently run on CI

Reply:
Yep, there are a number of files that are not consistent with black currently. `_version.py` and `frontend_semver.py` were generated by the `bump_version.py` script.